frontend/packages/knative-plugin/src/utils/__tests__/knative-topology-utils.spec.ts

```diff
@@ -116,3 +118,28 @@ describe('knative topology utils', () => {
     expect(filterRevisionsBaseOnTrafficStatus(mockService, mockRevisions)).toBeUndefined();
   });
 });
+
+describe('Knative Topology Utils', () => {
+  beforeAll(() => {
+    jest.spyOn(k8s, 'k8sUpdate').mockImplementation((model, data) => Promise.resolve({ data }));
+  });
+
+  afterAll(() => {
+    jest.restoreAllMocks();
+  });
+
+  it('should move sink to the target knServcice', (done) => {
+    createKnativeEventSourceSink(
+      MockKnativeResources.eventSourceCronjob.data[0],
+      MockKnativeResources.ksservices.data[0],
+    )
+      .then(({ data }) => {
+        expect(data.spec.sink.ref.name).toEqual('overlayimage');
+        expect(data.spec.sink.ref.kind).toEqual('service');
+        done();
+      })
+      .catch(() => {
+        done();
+      });
+  });
```

Comment:
Not a necessary change but I'd like to start moving to async await instead of promises.